examples/usb/main.cc

```diff
@@ -18,14 +18,47 @@
  *                                                                          *
  ****************************************************************************/
 
+/**
+ * sample client Python code to verify transmission correctness of this example program
+ *
+ * ```
+ * import serial
+ *
+ * ser = serial.Serial('/dev/ttyUSB0', baudrate=115200)
+ * some_str = 'this is my data!'
+ *
+ * for i in range(1000):
+ *     ser.write(some_str)
+ *     while ser.in_waiting < 3 * len(some_str):
+ *         continue
+ *     ret = ser.read_all()
+ *     assert ret == 3 * some_str
+ * ```
+ */
+
 #include "main.h"
 
 #include "bsp_usb.h"
+#include "bsp_gpio.h"
```

Comment:
I initially used the red light on the board to let me know if there are data transmissions. I deleted other codes, but I forgot this one since it was auto-collapsed by CLoin.